libs/components/toast/src/lib/modules/toast/toast.component.ts

```diff
@@ -45,13 +46,18 @@ export class SkyToastComponent implements OnInit, OnDestroy {
    * Specifies a `SkyToastType` type for the toast to determine the color and icon to display.
    */
   @Input()
-  public set toastType(value: SkyToastType) {
-    this._toastType = value;
+  public set toastType(value: SkyToastType | undefined) {
```

Comment:
I did a couple strict mode fixes in this file such as getting rid of the getter here and using the `OrDefault` convention. However, I did not strict mode everything including doing private private things as that was not in scope and there are a lot of violations in this component. I started to do them and it became too much for this unrelated fix and so I backed them out.